src/pipecat/services/together.py

```diff
@@ -58,18 +59,30 @@ def assistant(self) -> 'TogetherAssistantContextAggregator':
 class TogetherLLMService(LLMService):
     """This class implements inference with Together's Llama 3.1 models
     """
+    class InputParams(BaseModel):
+        frequency_penalty: Optional[float] = Field(default=None, ge=-2.0, le=2.0)
+        presence_penalty: Optional[float] = Field(default=None, ge=-2.0, le=2.0)
+        temperature: Optional[float] = Field(default=None, ge=0.0, le=1.0)
+        top_k: Optional[int] = Field(default=None, ge=0)
+        top_p: Optional[float] = Field(default=None, ge=0.0, le=1.0)
 
     def __init__(
             self,
             *,
             api_key: str,
             model: str = "meta-llama/Meta-Llama-3.1-8B-Instruct-Turbo",
             max_tokens: int = 4096,
+            params: InputParams = InputParams(),
```

Comment:
we should probably add `max_tokens` inside `InputParams` here as well.

Reply:
Same here.